contrib/agent.sh

```diff
@@ -13,6 +13,12 @@ tune_system_slice() {
   # Disable swap for system.slice
   _set 0 $FSROOT/sys/fs/cgroup/system.slice/memory.swap.max
 
+  # Disable swap for user.slice
+  _set 0 $FSROOT/sys/fs/cgroup/user.slice/memory.swap.max
+
+  # Disable swap for machine.slice
+  _set 0 $FSROOT/sys/fs/cgroup/machine.slice/memory.swap.max
+
```

Comment:
machine and user slice might not always exist.

Specifically `machine.slice`.
`user.slice` is only created on user-login iiuic.

We an do this, but then we should extend `_set` to check if the path exists.

If we wanted, then we could create systemd drop-ins instead - then they would be "boot persistent", and also applied to slices even if they are not there yet.

Reply:
@fabiand You are correct, we might not be in sync with the creation of these slices. 

Closing this PR in favor of drop-in creation

/close